iocore/net/quic/QUICLossDetector.cc

```diff
@@ -29,13 +29,14 @@
 #include "QUICEvents.h"
 #include "QUICDebugNames.h"
 #include "QUICFrameGenerator.h"
+#include "P_QUICNetVConnection.h"
```

Comment:
Do we need this? If yes, we probably need to introduce some interface. I'd never allow to include "P_QUICNetVConnection.h".

Reply:
Yep, miss that. Will remove